### ISSUE TITLE
Feature Request Option to add profiles to the Windows Start Menu

### ISSUE THREAD

**Issue by claudio4** [microsoft/terminal#2339]
# Description of the new feature/enhancement

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
This has little bits of #607 and #576 in it, but seems unique enough to warrant its own issue.

**Comment by kowalski7cc** [comment]
Maybe can be solved with ability to pin secondary tiles to start?
https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/secondary-tiles

**Comment by DHowett-MSFT** [comment]
Backlog, spec required, triaged.

**Comment by DHowett-MSFT** [comment]
Specifier should build on work already done on #576, because I am betting that it'll be along the same path!

**Comment by marveloo** [comment]
It would be great to be able to "Pin to Start" and "Pin to Taskbar" any profile we like. And then the profile opened profile via the Start (or Taskbar) tile should always open in a new window and the opened profile should become the default profile for this particular window, i.e. when I open a new tab in this window it should open the same profile.

**Comment by zadjii-msft** [comment]
For folks following this thread: A few releases ago, we added your profiles to the taskbar jumplist:
![image](https://user-images.githubusercontent.com/18356694/107390470-5a394d00-6abd-11eb-9bc3-7fc3c2737c7c.png)

which had the side-effect of displaying them in the start menu search as well:

![image](https://user-images.githubusercontent.com/18356694/107390630-7e952980-6abd-11eb-8681-ed273f174484.png)


I'm leaving this issue open to specifically track allowing you to pin the profiles as Start Menu tiles as well. Thanks!

**Comment by HendersonSC** [comment]
FWIW, a similar functionality is possible using shortcuts and batch files. Probably doesn't meet everyone's needs,

```
example.bat
wt -w 1 -p example_profile
```
Then its just right click, new->shortcut. Target of `cmd /c path/to/example.bat`, next, finish. Right click new shortcut pin to {start/taskbar}. In my use case, this has the additional benefit of allowing for complicated terminal tabs, new windows, and passing raw commands, i.e. set local scope path, open ssh tunnel, etc. The limit is seemingly what the wt command can handle.

**Comment by claudio4** [comment]
> FWIW, a similar functionality is possible using shortcuts and batch files. Probably doesn't meet everyone's needs,
> 
> ```
> example.bat
> wt -w 1 -p example_profile
> ```
> 
> Then its just right click, new->shortcut. Target of `cmd /c path/to/example.bat`, next, finish. Right click new shortcut pin to {start/taskbar}. In my use case, this has the additional benefit of allowing for complicated terminal tabs, new windows, and passing raw commands, i.e. set local scope path, open ssh tunnel, etc. The limit is seemingly what the wt command can handle.

Yeah, I'm been using my handmade shortcut since the WT allowed for opening specific profiles from the command line. But I still think there is value in the WT creating the shortcuts with just the click of button instead of us doing the manual process.

BTW, You can put the command directly in the shortcut and skip the batch file enterally.

**Comment by zadjii-msft** [comment]
FWIW, since there aren't tiles in Windows 11's start menu anymore, I'd reckon it's highly unlikely we (the Terminal team) get around to this ourselves. [The API](https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/secondary-tiles) doesn't seem terribly challenging to use, and it shouldn't be too hard to add a button to the Settings UI to activate this. If someone's passionate about adding the button for Windows 10, **we'd still accept the PR**. I just wanted to make sure everyone was aware of where this landed on the priorities.
